datafusion/substrait/src/logical_plan/consumer.rs

```diff
@@ -1586,9 +1629,12 @@ fn next_struct_field_name(
     }
 }
 
-fn from_substrait_named_struct(
+/// Convert Substrait NamedStruct to DataFusion DFSchemaRef
+pub fn from_substrait_named_struct(
```

Comment:
While this is generally useful, I could see a case for keeping this internal only for now. That was actually what I tried to do originally, but I'm new-ish to Rust and couldn't figure out a good way to expose this for use in integration tests only.

Reply:
Weird. Well, I guess it's okay to be public too